go.mod

```diff
@@ -1,44 +1,44 @@
 module github.com/openstack-k8s-operators/heat-operator
 
-go 1.19
+go 1.20
 
 replace github.com/openstack-k8s-operators/heat-operator/api => ./api
 
 require (
 	github.com/go-logr/logr v1.4.1
 	github.com/google/uuid v1.6.0
-	github.com/onsi/ginkgo/v2 v2.13.2
+	github.com/onsi/ginkgo/v2 v2.14.0
 	github.com/onsi/gomega v1.30.0
-	github.com/openstack-k8s-operators/heat-operator/api v0.1.1-0.20230828054057-36837cde8504
+	github.com/openstack-k8s-operators/heat-operator/api v0.0.0-00010101000000-000000000000
 	github.com/openstack-k8s-operators/infra-operator/apis v0.3.1-0.20240104150635-c4ffc51e0752
 	github.com/openstack-k8s-operators/keystone-operator/api v0.3.1-0.20240104144437-5355d932c316
 	github.com/openstack-k8s-operators/lib-common/modules/common v0.3.1-0.20240110131857-e70e1dec4d14
 	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.3.1-0.20240106101723-5f7aa263457f
 	github.com/openstack-k8s-operators/lib-common/modules/test v0.3.1-0.20240106101723-5f7aa263457f
 	github.com/openstack-k8s-operators/mariadb-operator/api v0.3.1-0.20240208072109-4447f245e487
-	k8s.io/api v0.26.13
-	k8s.io/apimachinery v0.26.13
-	k8s.io/client-go v0.26.13
+	k8s.io/api v0.29.1
+	k8s.io/apimachinery v0.29.1
+	k8s.io/client-go v0.29.1
 	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
-	sigs.k8s.io/controller-runtime v0.14.7
+	sigs.k8s.io/controller-runtime v0.17.0
```

Comment:
I this should be 0.16

Reply:
I think it's because I was just trying to downgrade controller-runtime which didn't auto downgrade the k8s.io dependencies, then mod tidy puts this back.

This seems more reliable now:
```
go1.20.3 get -u k8s.io/api@v0.28.3
```

Which also downgrades controller-runtime to 0.16.4.